.drone.yml

```diff
@@ -1,26 +1,40 @@
 matrix:
   include:
-    - DOCKER_RUBY_VERSION: 1.9.3
-      RUBY_IMAGE_TAG: 1.9.3-2
-
     - DOCKER_RUBY_VERSION: 2.2
-      RUBY_IMAGE_TAG: 2.2-2
+      RUBY_IMAGE_TAG: 2.2-latest
+
+    - DOCKER_RUBY_VERSION: 2.3
+      RUBY_IMAGE_TAG: 2.3-latest
 
 build:
-  image: abakpress/dind:2
+  image: abakpress/dind-testing
   privileged: true
   volumes:
     - /home/data/drone/images:/images
     - /home/data/drone/gems:/bundle
+    - /home/data/drone/key_cache:/ssh_keys
   environment:
     - COMPOSE_FILE_EXT=drone
+    - RAILS_ENV=test
+    - RUBY_IMAGE_TAG=2.2-latest
   commands:
     - wrapdocker docker -v
 
-    - if [ ! -e /images/ruby_$RUBY_IMAGE_TAG.tar ]; then docker pull abakpress/ruby:$RUBY_IMAGE_TAG; docker save abakpress/ruby:$RUBY_IMAGE_TAG > /images/ruby_$RUBY_IMAGE_TAG.tar; fi
-
-    - docker load -i /images/ssh-agent.tar
-    - docker load -i /images/ruby_$RUBY_IMAGE_TAG.tar
+    - fetch-images
+      --image whilp/ssh-agent
+      --image abakpress/ruby-app:$RUBY_IMAGE_TAG
 
+    - dip ssh add -T -v /ssh_keys -k /ssh_keys/id_rsa
```

Comment:
why we need ssh key here? I thought it's open-source no?

Reply:
copy-paste

Reply:
fixed